IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedBldgElements/Entities/IfcFillElement/DocEntity.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcFillElement" UniqueId="ac535580-9989-4b4f-9e24-e1699ef0a2cf" BaseDefinition="IfcBuiltElement" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="PredefinedType" UniqueId="6fbf8ece-aa38-428b-bfdc-bca7d3978be5" DefinedType="IfcFillElementTypeEnum" AttributeFlags="1" />
+	</Attributes>
+	<WhereRules>
+		<DocWhereRule Name="CorrectPredefinedType" UniqueId="ede8b188-24c4-4563-93da-eb628145dd99">
```

Comment:
Please check the PredefinedType rules. I have the impression that we changed them at all other occassions.

Reply:
In this case, I think that this is fine since we also have an IfcFillElementType (which was not the case in all instances where we changed). @SergejMuhic: can you please check also before merging?

Reply:
Correct. These should be as they are due to `IfcTypeObject` being defined for `IfcFillElement`.